### DISCUSSION TITLE
Already using NPM - can I bypass nginx?

### DISCUSSION THREAD

**Discussion by Arcau** [amidaware/tacticalrmm#1862] [Q&A]
Fairly new to this stuff - I have a server running NPM handling all my other port forwards for 443 and 80. 
Sites and things being passed through there to the various hosts as needed. 
But when attempting to add in the tactical vm to it - it fails and can not ping from outside. 

I was wondering if there was some way that I am just clearly missing to get my current NPM setup to pass through to Tactical nicely so it works. 

so NPM hosted on 10.100.10.62
ports 80 and 443 go there 

it then routes everything where it needs to go to various dockers etc 
but if I point it to tactical it doesnt work. 
https / 10.100.10.98 / 443
It will "load" the login page if clicking the link from NPM locally - but wont resolve if URL put in. 
and unable to login. 

If I turn off my NPM and pass through directly 443 traffic to tactical VM it works and I can login to the RMM page no issues. 

Just wondering if there is a nice way to make this work for me

**Comment by silversword411** [comment]
Start here https://docs.tacticalrmm.com/unsupported_guidelines/

**Reply by Arcau** [reply]
Thanks I have followed that guide but am now getting 502 bad gateway :( 
Following the mesh central part and added in the IP of the NPM instance but still no good. 
I am pretty lost at this point so any help is appreciated. 

"WANonly": true,
    "minify": 1,
    "port": 4430,
    "aliasPort": 443,
    "redirPort": 800,
    "allowLoginToken": true,
    "allowFraming": true,
    "agentPing": 35,
    "allowHighQualityDesktop": true,
    "tlsOffload": "10.100.10.62",
    "trustedProxy": "10.100.10.62",
    "agentCoreDump": false,
    "compression": true,
    "wsCompression": true,
    "agentWsCompression": true,
    "maxInvalidLogin": { "time": 5, "count": 5, "coolofftime": 30 },

**Reply by silversword411** [reply]
https://docs.tacticalrmm.com/support/#commercial-support
